### ISSUE TITLE
putIterationsPerSec before added to Buffer

### ISSUE THREAD

**Issue by Martin99y** [zhoubolei/TRN-pytorch#87]
Hi i am really a beginner so i am not sure if i got this right,
but inn line 136 of fps_dem_trn.py  putIterationsPerSec is called which will draw the class label on this image. But the same image is also used to predict the labels later on because it is put into the buffer. Why not change line 140 and 136 and then instead of calling crop_img = frame[0:720, 0:720] do crop_img = frame.copy(). Also line 144 will be needed to be changed to cv2.imshow("Frame", frame). And thanks for the great repo its really cool to see the model working in realtime.

Regards Martin Pflaum

**Comment by Martin99y** [comment]
Its also a bit strange that there is no regulation in which frequence the frames are put into the buffer. so accuracy changes depending on how fast or slow the pc is. Does some one know the original frame rate the jester pretrained model was trained?
regards martin
